### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'has_key' - Python 3 removed has_key()

### ISSUE THREAD

**Issue by quatauta** [gapan/xdgmenumaker#11]
With Python 3, line 369 `if category_menus.has_key(candidate):` in clean_up_categories() throws AttributeError: 'dict' object has no attribute 'has_key'.

```
Traceback (most recent call last):
  File "xdgmenumaker", line 858, in <module>
    main(sys.argv[1:])
  File "xdgmenumaker", line 225, in main
    fluxbox()
  File "xdgmenumaker", line 535, in fluxbox
    for menu_category in menu():
  File "xdgmenumaker", line 466, in menu
    entry = get_entry_info(desktopfile, ico_paths=ico_paths)
  File "xdgmenumaker", line 414, in get_entry_info
    category = clean_up_categories(categories)
  File "xdgmenumaker", line 369, in clean_up_categories
    if category_menus.has_key(candidate):
AttributeError: 'dict' object has no attribute 'has_key'
```

dict.has_key() was removed in Python 3: Removed. dict.has_key() – use the in operator instead. (https://docs.python.org/3.1/whatsnew/3.0.html#builtins)

**Comment by quatauta** [comment]
```
--- xdgmenumaker.a       2017-07-19 20:02:17.748493056 +0200
+++ xdgmenumaker.b        2017-07-19 20:05:16.622386461 +0200
@@ -366,7 +366,7 @@
     }
     category = other
     for candidate in categories:
-        if category_menus.has_key(candidate):
+        if candidate in category_menus:
             category = category_menus.get(candidate)
             break
     return category
```

**Comment by gapan** [comment]
Fixed with commit dcc48c66110f456dd12780bc13d21260445a323e
